src/heitech.zer0mqXt.core/Main/Socket.cs

```diff
@@ -29,6 +29,13 @@ public void Dispose()
             _sendReceive.Dispose();
         }
 
+        // we need to use Bind for a publisher so Subscribers can connect to it.
+        // this is due to the mechanisms integral to Zer0MqItself
```

Comment:
A bit about styling on the comment:
Consistency: First line finished with a `.` - but second line does not.
Typo: Please insert a space between "Zer0Mq" and "itself" - and lowercase "itself".